docs/docs/vecsim-hybrid_queries_examples.ipynb

```diff
@@ -0,0 +1,320 @@
+     ]
+    }
+   ],
+   "source": [
+    "# Give me the top 10 reviews on action movies similar to mine\n",
+    "\n",
+    "q = Query(f'(@{genre_field_name}:{{action}})=>[KNN 10 @{vector_field_name} $vec_param AS dist]').sort_by('dist')\n",
+    "res = redis_conn.ft().search(q, query_params = {'vec_param': query_vector.tobytes()})\n",
+    "\n",
+    "print_results(res)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "d2f48df5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "got 10 doc ids:  [5126, 6268, 5390, 5085, 6741, 6251, 5239, 5487, 5194, 5595]\n",
+      "\n",
+      "distances:  [14.7560760961, 15.2173650941, 15.7935849617, 15.8196561477, 15.8495724098, 15.8533288875, 16.0165456443, 16.0417755318, 16.0750138339, 16.2356399735]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Give me the top 10 reviews on action movies similar to mine that got ratings between 5-7.\n",
+    "# (ids 5000-7000)\n",
+    "\n",
+    "q = Query(f'(@{genre_field_name}:{{action}} @{rating_field_name}:[5 7])=>[KNN 10 @{vector_field_name} $vec_param AS dist]').sort_by('dist')\n",
+    "res = redis_conn.ft().search(q, query_params = {'vec_param': query_vector.tobytes()})\n",
+    "\n",
+    "print_results(res)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "b2284b3b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "got 10 doc ids:  [5390, 5085, 5595, 6695, 6285, 5765, 6595, 5795, 5790, 5550]\n",
+      "\n",
+      "distances:  [15.7935849617, 15.8196561477, 16.2356399735, 16.4198694829, 16.4199152798, 16.4874357724, 16.59035834, 16.657459116, 16.6816978817, 16.786226798]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Give me the top 10 reviews on spiderman movie that are similar to mine and got ratings between 5-7.\n",
+    "#(ids 5000-7000 divided by 5)\n",
+    "\n",
+    "q = Query(f'(@{title_field_name}:spiderman @{rating_field_name}:[5 7])=>[KNN 10 @{vector_field_name} $vec_param AS dist]').sort_by('dist')\n",
+    "res = redis_conn.ft().search(q, query_params = {'vec_param': query_vector.tobytes()})\n",
+    "\n",
+    "print_results(res)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "fb3dce42",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "got 10 doc ids:  [9386, 83, 5126, 9572, 3492, 6268, 3949, 4437, 1057, 557]\n",
+      "\n",
+      "distances:  [14.5484484676, 14.7082952384, 14.7560760961, 14.8371418493, 14.9124708649, 15.2173650941, 15.3307324484, 15.3791827069, 15.488778035, 15.4977867241]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Give me the top 10 reviews on movies which aren't spiderman that are similar to mine.\n",
```

Comment:
```suggestion
    "# Give me the top 10 reviews on movies that aren't Spiderman that are similar to mine.\n",
```

Reply:
This query description is a little confusing to me. What is similar exactly?

Reply:
The reviews itself are semantically similar. That's the purpose of using vector embedding - we expect that similar items will be represented by close vectors....